pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -383,7 +383,12 @@ func streamPathToBuild(git git.Repository, in io.Reader, out io.Writer, client o
 			return nil, err
 		}
 		if stat.IsDir() {
-			info, gitErr := gitRefInfo(git, clean, "HEAD")
+			commit := "HEAD"
+			if len(options.Commit) > 0 {
+				commit = options.Commit
+			}
+			fmt.Fprintf(out, "Uploading %q at commit %q as binary input for the build ...\n", clean, commit)
```

Comment:
@smarterclayton I shortened this message because it ruins the feng-shui of my terminal :-) I think from the message it is clear it is uploading (git) repo (commit).